pkg/minishift/provisioner/buildroot_provisioner.go

```diff
@@ -83,28 +83,40 @@ func (p *BuildrootProvisioner) Provision(swarmOptions swarm.Options, authOptions
 	p.AuthOptions = authOptions
 	p.EngineOptions = engineOptions
 
-	log.Debugf("setting hostname %q", p.Driver.GetMachineName())
+	log.Info("\n   Setting hostname ... ")
 	if err := p.SetHostname(p.Driver.GetMachineName()); err != nil {
+		log.Info("FAIL")
 		return err
+	} else {
+		log.Info("OK")
 	}
 
-	p.AuthOptions = setRemoteAuthOptions(p)
-	log.Debugf("set auth options %+v", p.AuthOptions)
+	if err := makeDockerOptionsDir(p); err != nil {
+		return err
+	}
+
+	dockerCfg, err := p.GenerateDockerOptions(engine.DefaultPort)
```

Comment:
Line 98-107 seems to be repeated in `provision.ConfigureAuth` function.
See https://github.com/minishift/minishift/blob/52b35d0a4f5119bb9843d5712d7e7fd1e5f6dd70/pkg/minishift/provisioner/utils.go#L252-L261

Reply:
If you check this PR, I removed this function because we were using it as minikube does and it was not working for us anymore.